llvm/test/CodeGen/AMDGPU/call-argument-types.ll

```diff
@@ -124,8 +124,9 @@ define amdgpu_kernel void @test_call_external_void_func_i1_imm() #0 {
 ; GFX11-NEXT:    s_mov_b64 s[6:7], s[0:1]
 ; GFX11-NEXT:    s_mov_b32 s32, 0
 ; GFX11-NEXT:    s_getpc_b64 s[2:3]
-; GFX11-NEXT:    s_add_u32 s2, s2, external_void_func_i1@rel32@lo+4
-; GFX11-NEXT:    s_addc_u32 s3, s3, external_void_func_i1@rel32@hi+12
+; GFX11-NEXT:    s_delay_alu instid0(SALU_CYCLE_1)
```

Comment:
Open question: should `AMDGPUInsertDelayAlu` have updated this `s_delay_alu` instruction (which is inside a bundle) instead of adding the new `s_delay_alu` instruction on line 130 right after the bundle?